extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -217,7 +215,7 @@
 	'communitypage-entry-button' => 'Leg los',
 	'communitypage-tasks-title' => 'Aufgaben',
 	'communitypage-entry-tasks' => 'Hilf mit bei der Bearbeitung dieser Community',
-	'communitypage-tasks-header-welcome' => 'Hilf uns dabei, {{SITENAME}} zu erweitern!',
+	'communitypage-tasks-header-welcome' => 'Herzlich Willkommen $1 Fan!',
```

Comment:
This change will break existing functionality and will show `Herzlich Willkommen Fan!`